plugins/config/lib/configService.js

```diff
@@ -2264,6 +2337,21 @@ function getPluginConfiguration(identifier, pluginLocation, serverSettings,produ
 };
 exports.getPluginConfiguration = getPluginConfiguration;
 
+function getAllowedPlugins(scope, options, username) {
```

Comment:
You dont need SCOPE as a variable here because, if username == null|undefined, then SCOPE=INSTANCE, else SCOPE=USER